TestSwitchApi/TestSwitchApi/Models/Request/PageRequest.cs

```diff
@@ -4,5 +4,7 @@ public class PageRequest
     {
             public int Page { get; set; } = 1;
             public int PageSize { get; set; } = 10;
+
+            public string SessionId { get; }
```

Comment:
Can you just make this a `Guid`? 
Does that avoid the converting to and from a string?

Reply:
which conversions are you talking about? I found the one which converts the DB Guid to a string, but that's to compare it to the header ID which is a string

Reply:
I think that one above relates to something else (not even sure if it's used for anything)

Reply:
oh right - sure yeah that's fine then